src/screen.rs

```diff
@@ -115,9 +122,9 @@ impl Default for Screen {
 impl Screen {
     fn help(&mut self) {
         self.cleanup();
-        print!("{}{}{}\n", cursor::Goto(1, 1), clear::All, self.config);
+        println!("{}{}{}", cursor::Goto(1, 1), clear::All, self.config);
```

Comment:
`println` acquires a Mutex and writes the line synchronously, but `print` allows things to buffer up more before flushing (at least last time I looked into this). If that's no longer the case, then this is fine, but please double check and keep it as a `print` if the lock is acquired.

Reply:
I just looked at the source code for both and they use the same function (`io::_print`) to do it, except println uses a macro that adds a newline at the end.